src/solver_info.jl

```diff
@@ -4,18 +4,25 @@ export can_solve_mip, can_solve_socp, can_solve_sdp, can_solve_exp
 export set_default_solver, get_default_solver, isinstalled
 
 function set_default_solver(solver::MathProgBase.AbstractMathProgSolver)
+    Base.depwarn("Default solvers are deprecated and will not be provided in a future " *
+                 "release.\nLoad any necessary solvers manually.", :set_default_solver)
+
     global DEFAULT_SOLVER
     DEFAULT_SOLVER = solver
 end
 
 function get_default_solver()
+    Base.depwarn("Default solvers are deprecated and will not be provided in a future " *
+                 "release.\nLoad any necessary solvers manually.", :get_default_solver)
```

Comment:
I would give a helpful message that you can run, e.g., `using SCS` if you have it installed. I think it's important to be excessively helpful here, just as the error below is.

Reply:
How do folks feel about this warning?
```
Base.depwarn(
        "Default solvers are deprecated and will not be provided in a future release.
        You can still manually load and use installed solvers such as ECOS by running:
        using ECOS
        p = minimize(...);
        solve!(p, ECOSSolver())",
        :get_default_solver
 )
```